### PR TITLE
ADT service implementation enhancements

### PR DESCRIPTION
**Pull request by jecihjoy**

### CODE REVIEW

File: app/app.ts [hunk 1]
```diff
@@ -8,8 +8,8 @@ import { adtRoutes } from "./api/adt-routes";
 import config from "./config";
 const init = async () => {
   const server: Server = new Server({
-    port: 3000,
-    host: config.server,
+    port: 9000,
+    host: "localhost",
```
Comment (drizzentic) [inline app/app.ts:12]: Why use a static value instead of the configs variables?

File: app/services/prescription.ts [hunk 2]
```diff
@@ -1,43 +1,168 @@
...
+          (ob: any) =>
+            ob.concept.uuid === "a899cf5e-1350-11df-a1f1-0026b9348838"
+        );
+        break;
+      /** CHANGE REGIMEN, CHANGE DOSE, CHANGE FORMULATION, DRUG SUBSTITUTION, START ARVS, RESTART*/
+      case "a89b7c50-1350-11df-a1f1-0026b9348838":
+      case "a898c938-1350-11df-a1f1-0026b9348838":
+      case "a89b7ae8-1350-11df-a1f1-0026b9348838":
+      case "a8a00158-1350-11df-a1f1-0026b9348838":
+      case "a89b77aa-1350-11df-a1f1-0026b9348838":
```
Comment (drizzentic) [inline app/services/prescription.ts:57]: What does this case expression do?

File: app/subscribers/prescription.ts [hunk 3]
```diff
@@ -2,34 +2,51 @@ import { EventSubscriber, On } from "event-dispatch";
...
     let provider: EPrescription.OrderingPhysician = await loadProviderData(
-      amrsCCC,
+      savedAmrsOrders[0].orderer.uuid,
       amrsCon
     );
+    let encounter = await loadEncounterData(savedAmrsOrders[0].encounter.uuid);
     const data = new ADTRESTClient("");
     const regimenLoader = new RegimenLoader();
-    const regimen = regimenLoader.getRegimenCode(patients.start_regimen)[0];
+    const regimen = regimenLoader.getRegimenCode("3TC + NVP + AZT")[0];
```
Comment (drizzentic) [inline app/subscribers/prescription.ts:26]: Why do we have this hardcoded?

**Review by drizzentic — APPROVED** [review]
Good stuff!

**Comment by jecihjoy** [comment]
@drizzentic would you also merge these because I don't have merge rights

**Pull request merged**